webapp/IronCalc/src/components/workbook.tsx

```diff
@@ -143,16 +141,10 @@ const Workbook = (props: { model: Model; workbookState: WorkbookState }) => {
     // FIXME: This is so that the cursor indicates there are styles to be pasted
     const el = rootRef.current?.getElementsByClassName("sheet-container")[0];
     if (el) {
+      // Taken from lucide icons: <PaintRoller /> and rotated.
```

Comment:
Hi @dg-ac , this was a tough one but the old code was using react-dom and increasing the build by 300K!